src/domain/account/errors/email-already-verified.error.ts

```diff
@@ -0,0 +1,10 @@
+export class EmailAlreadyVerifiedError extends Error {
+  readonly signer: string;
+
+  constructor(args: { chainId: string; safeAddress: string; signer: string }) {
+    super(
+      `The email address is already verified. chainId=${args.chainId}, safeAddress=${args.safeAddress}, signer=${args.signer}`,
+    );
+    this.signer = args.signer;
```

Comment:
```suggestion
```

Reply:
The property is used in the `EmailAlreadyVerifiedExceptionFilter` so we need to store it or?

Reply:
Indeed 🦅